ee/hogai/videos/utils.py

```diff
@@ -0,0 +1,17 @@
+import math
+from io import BytesIO
+
+import structlog
+from pymediainfo import MediaInfo
+
+logger = structlog.get_logger(__name__)
+
+
+def get_video_duration_s(video_bytes: bytes) -> float:
+    """Extract duration in milliseconds from video bytes to understand when the export UI finished rendering"""
+    media_info = MediaInfo.parse(BytesIO(video_bytes))
+    for track in media_info.tracks:
+        if track.track_type == "General":
+            # Convert ms to seconds, ceil to avoid grey "not-rendered" frames at the start
+            return math.ceil(track.duration / 1000.0)
```

Comment:
Q @sortafreel: I copied this verbatim from the existing code, but I'm not sure if it's actually intended to ceil here? Seems like if we wanted to skip, we'd floor. And either way, this should not be needed, as we have a more intentional `SESSION_VIDEO_RENDERING_DELAY` offset